web/src/features/charts/bar-breakdown/utils.ts

```diff
@@ -192,3 +193,14 @@ export const getExchangesToDisplay = (
     (exchangeZoneKey) => !exchangeZoneKeysToRemove.has(exchangeZoneKey)
   );
 };
+
+export const getHeaderHeight = () => {
+  const [headerHeight, setHeaderHeight] = useState<number>(0);
+
+  useEffect(() => {
+    const height = document.querySelector('header').offsetHeight;
+    setHeaderHeight(height * 1.1);
+  }, [window.innerWidth, window.innerHeight]);
+
+  return headerHeight;
+};
```

Comment:
```suggestion
export const getHeaderHeight = () => {
  const [headerHeight, setHeaderHeight] = useState<number>(0);

  useEffect(() => {
    const height = document.querySelector('header')?.offsetHeight ?? 0;
    setHeaderHeight(height * 1.1);
  }, [window.innerWidth, window.innerHeight]);

  return headerHeight;
};
```
This is to fix a type issue since `document.querySelector()` can return `Null`.